src/components/app/header/header.js

```diff
@@ -9,29 +10,66 @@ import ProfileImagePath from 'assets/images/profile.png';
 
 import './header.css';
 
+const docsDropdown = 'docsDropdown';
+const profileDropdown = 'profileDropdown';
+
+const parentHasClass = (element, ...searchClasses) => {
+  if (
+    typeof element.className === 'string' &&
+    element.className
+      .split(' ')
+      .some(classname =>
+        searchClasses.some(searchClass => searchClass === classname)
+      )
+  ) return true;
+  return element.parentNode && parentHasClass(element.parentNode, ...searchClasses);
+};
+
+const docLinks = [
+  {
+    translationId: 'header.getStarted',
+    url: 'https://docs.microsoft.com/en-us/azure/iot-suite/iot-suite-remote-monitoring-monitor'
+  },
+  {
+    translationId: 'header.documentation',
+    url: 'https://docs.microsoft.com/en-us/azure/iot-suite'
+  },
+  {
+    translationId: 'header.sendSuggestion',
+    url: 'https://feedback.azure.com/forums/321918-azure-iot'
```

Comment:
Is there a specific link for azure iot solutions?

Reply:
Should these URLs be in config so a "code" change isn't needed?

Reply:
I don't know. These are the links from the spec that I got.